server/views/components/new-site-notice/index.njk

```diff
@@ -5,7 +5,7 @@
         <div class="new-site-notice__inner">
           <p class="new-site-notice__msg">
             <span class="new-site-notice__device-ico">{% icon 'other/devices' %}</span>
-            <span class="new-site-notice__text">Welcome to the new reading experience for stories and articles. <span class="new-site-notice__cta"><a href="/articles/next-wellcome-website">Find out more</a>.</span></span>
+            <span class="new-site-notice__text">Welcome to the new reading experience for stories and articles. {% if showLink %}<span class="new-site-notice__cta"><a href="/">Find out more</a>.</span>{% endif %}</span>
```

Comment:
I'd probably break the line down at the inner span, as it's a bit long. 👍  otherwise